src/modules/editor/SoundEditor/SoundEditor.css

```diff
@@ -13,20 +14,100 @@
     display: flex;
     flex-direction: column;
     flex-wrap: nowrap;
-    margin: 10px 0;
     /* align-items removed to avoid clipping */
 }
 
 .grid {
     width: max-content;
     display: grid;
-    gap: 1px;
-    background-color: black;
+    margin-top: 1em;
+    row-gap: 2px;
+    background-color: #537D8D;
     box-sizing: border-box;
+    border : 3px solid #537D8D;
+
+
+    .cell {
+        background-color: #3a5863;
+        color: transparent;
+        user-select: none;
+    }
+
+    .cell:hover {
+        background-color: #2a3c45;
+    }
+
+    .cell.selected {
+        background-color: #2a3c45;
+        color: black; 
+    }
 }
 
-.grid-cell {
-    background-color: white;
-    border: 1px solid gray;
-    box-sizing: border-box;
+
+.button {
```

Comment:
I believe this shouldn't be here at all, but rather part of a component **IF** this is a generic UI button indeed. Otherwise, rename it so that it is a specialised button for the sound editor.